data/records/cms-tools-ispy.json

```diff
@@ -115,7 +115,7 @@
       "CMS"
     ],
     "license": {
-      "attribution": "BSD License"
+      "attribution": "BSD-3-Clause"
```

Comment:
No further information on which exact BSD license is used

Reply:
Looking into the source code repository linked in that record: https://github.com/cms-outreach/ispy-webgl/blob/master/LICENSE I see that the license is actually MIT now, not BSD.